### DISCUSSION TITLE
小米cr660x无法运行？

### DISCUSSION THREAD

**Discussion by cicai1631** [messense/aliyundrive-webdav#120] [Q&A]
openclash 本地Dns解析开启，
adguard home上游dnsmaq开启

手动安装aliyun-webdev 三个ipk文件
主程序ipk，Luci ，还有一个zh-cn ipk刷新后，可以在服务中看见界面就是无法运行？什么问题？
tokens没问题

之前因为atv观看阿里云dev不稳定，所以重新安装op系统。

**Comment by messense** [comment]
请提供日志。

**Reply by cicai1631** [reply]
已经升级到0.5.3
日志空白，什么都没有

**Reply by messense** [reply]
登录路由器命令执行 aliyundrive-webdav -r refresh_token 看看输出。

**Reply by cicai1631** [reply]
> 登录路由器命令执行 aliyundrive-webdav -r refresh_token 看看输出。

什么都没返回，
root@ImmortalWrt:~#

**Reply by messense** [reply]
那就很奇怪了，要么报个错，要么有些输出。可能你的 OpenWrt 系统有些问题？

**Reply by cicai1631** [reply]
> 那就很奇怪了，要么报个错，要么有些输出。可能你的 OpenWrt 系统有些问题？

Powered by LuCI openwrt-18.06-k5.4 branch (git-21.170.50917-05a4f95) / ArgonTheme v1.7.2 / ImmortalWrt 18.06-SNAPSHOT r10213-8c7d866ba

**Reply by messense** [reply]
建议尝试[命令行](https://github.com/messense/aliyundrive-webdav#%E5%91%BD%E4%BB%A4%E8%A1%8C%E7%94%A8%E6%B3%95)跑通或者复现并给出输出/日志.

**Reply by cicai1631** [reply]
> 建议尝试[命令行](https://github.com/messense/aliyundrive-webdav#%E5%91%BD%E4%BB%A4%E8%A1%8C%E7%94%A8%E6%B3%95)跑通或者复现并给出输出/日志.

新手，添加什么命令可以？
